client/src/app/employee/employee-list/state/employee-list.service.ts

```diff
@@ -1,17 +1,18 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { ToastrService } from 'ngx-toastr';
 import { of } from 'rxjs';
 import { catchError } from 'rxjs/operators';
-import { EnvService } from 'src/app/shared/env.service';
-import {
-  ListResponse,
-  mapListResponseToData,
-} from 'src/app/shared/list-response';
-import { Employee } from '../../employee-detail/state/employee.model';
+
+import { ToastrService } from 'ngx-toastr';
+
+import { EmployeeDetail } from '@employee/employee-detail/state/employee.model';
+
 import { EmployeeCreate, EmployeeListItem } from './employee-list.model';
 import { EmployeeListStore } from './employee-list.store';
 
+import { EnvService } from 'src/app/shared/env.service';
+import { ListResponse, mapListResponseToData } from 'src/app/shared/list-response';
```

Comment:
I think these absolute imports were from the solution style `tsconfig.json` that Angular has now reverted from using.

We have the `@shared` alias so these can be changed to `@shared/env.service` and `@shared/list-response`

Reply:
fixed these across the app